src/odemis/driver/keysight.py

```diff
@@ -0,0 +1,333 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 29 Apr 2024
+
+@author: Canberk Akin
+
+Copyright © 2024 Canberk Akin, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+
+import logging
+import re
+import time
+import socket
+
+from odemis import model
+from odemis.model import HwError
+from odemis.util import to_str_escape
+from typing import List, Tuple
+
+
+class TrueForm(model.Emitter):
+    def __init__(self, name, role, address, channels: List[int],
+                 limits: List[Tuple[float, float]], **kwargs):
+        """ Initializes the Keysight 33600 series Arbitrary Waveform Generator.
+        :param name: (str) as in Odemis
+        :param role: (str) as in Odemis
+        :param address: # “fake” (to start the simulator) or an IP address
+        :param channels: # Channels which generate the waveform
+        :param limits: # min/max V for each channel. if min>max, the polarity of the channel is inverted.
+        """
+        delay = "DEFAULT VALUE"
+        self.delay = model.FloatContinuous(delay, setter=self._setTimeRange, unit="s")
+
+        dutyCycle = "DEFAULT VALUE"
+        self.dutyCycle = model.FloatContinuous(dutyCycle, setter=self.setDutyCycle, unit="s")
+
+        period = 4e-6  # default value
```

Comment:
"default value" as a comment is not helpful. You could instead indicate that it happens to be the standard value we use in our first system using this. You can also indicate the value in MHz.
That said, I thought we will use 40MHz by default, so that's odd.

Reply:
Will update the comment and make it informative and smarter. 

The frequency is in Hertz. So the default value of the period is indeed missing a 0.